composer.json

```diff
@@ -32,7 +32,8 @@
     "cweagans/composer-patches": "^1.6.5",
     "drupal-composer/drupal-scaffold": "^2.5",
     "apigee/apigee_devportal_kickstart": "^1.0.0",
-    "php": "^7.1"
+    "php": "^7.1",
+    "drush/drush": "^9.0.0"
```

Comment:
One thing to keep in mind on this, is that Pantheon still recommends Drush 8 in [their Composer project](https://github.com/pantheon-systems/example-drops-8-composer/blob/master/composer.json#L22), and in their [documentation](https://pantheon.io/docs/drush-versions/). They did make Drush 9 available recently, but there are [caveats](https://pantheon.io/blog/drush-940-release-available-pantheon). I tried to use it with their drops-8 project and CircleCI recently (which is a very [common setup](https://pantheon.io/docs/guides/build-tools/)), and it broke the build, and I had to revert back to Drush 8.

I've also experienced issues where having a "site drush" (essentially what this PR does) caused issues with Lando. The Lando Drupal 8/Pantheon recipe [installs Drush 8](https://docs.devwithlando.io/tutorials/pantheon.html#using-drush), as "global drush".

> By default our Pantheon recipe will globally install the latest version of Drush 8 

> Because of how Lando's php service sets up its PATH this means that if you have indeed installed Drush on your own via composer Lando will use yours over its own. Said more explicitly: if you've required drush via composer in your application then this recipe will use your drush and not the one you've specified in this recipes config.

Long story short: this _might_ be a can of worms, unless we add some documentation.

Reply:
@jacine has a point with Pantheon, as I've also ran into a couple of issues myself in the past with drush 9, and ended up reverting to 8, so I think it would be safest to require drush 8. Updating PR with it.